Sources/Paywalls/Components/Common/Dimension.swift

```diff
@@ -18,10 +18,10 @@ import Foundation
 
 public extension PaywallComponent {
 
-    enum Dimension: Codable, Sendable, Hashable {
+    @frozen enum Dimension: Codable, Sendable, Hashable {
```

Comment:
I must say I'm not a big fan of this `@frozen` annotation... While it does make sense, it seems it might cause us some issues down the line. For this enum I don't foresee new values coming anytime soon but well... In any case, if this is the recommended approach, seems we just will need to be thinking forward a lot for these APIs.

Reply:
I will just remove them for now 😇